theme/src/css/_zendesk-homepage.scss

```diff
@@ -5,70 +5,94 @@
 		}
 
 		.announcement-body {
-			margin-bottom: 3.5rem;
-			margin-left: 3rem;
+			margin-bottom: 3rem;
+			margin-left: 3.25rem;
+			margin-top: $margin-default;
 		}
 
 		.announcement-heading {
 			align-items: center;
 			display: flex;
+			margin-top: $margin-default;
 		}
 
 		.announcement-icon {
-			height: 2rem;
-			margin-right: 1rem;
-			width: 2rem;
+			height: 2.5rem;
+			margin-right: $margin-sm;
+			width: 2.5rem;
+		}
+
+		.article-list-item a {
+			color: $color-body;
 		}
 
 		.cta {
-			border: 1px solid $color-border;
+			background-color: $color-grey;
 			border-radius: $border-radius;
-			box-shadow: 0 6px 15px -7px rgba(9, 16, 29, 0.05);
-			padding: $padding-md $padding-default $padding-default;
+			margin-bottom: 4rem;
+			padding: 3rem $padding-default;
+			text-align: center;
+			width: 100%;
+
+			.btn-outline-primary {
+				margin-top: $margin-default;
+			}
 		}
 	}
 
 	.banner {
-		align-items: center;
-		display: flex;
-		min-height: 26.5rem;
-
-		.banner-img-container {
-			.img {
-				margin-top: 3.125rem;
-				max-height: 100%;
-				max-width: 100%;
-			}
+		background: linear-gradient(107.81deg, #113B87 0.45%, rgba(17, 59, 140, 1) 28.91%, rgba(16, 56, 124, 1) 100%);
+		background-color: $color-brand;
+		background-size: cover;
+		display: block;
+		margin-top: $navbar-height;
+		min-height: 414px;
+		
+		#query {
+			background-color: rgba(255, 255, 255, 0.2);
+			border: 0;
+			border-left: 3px solid #FDCC4A;
```

Comment:
Should we make this a color SASS variable like the other colors?

Reply:
I wasn't sure about this one, because it's a new yellow color that's only being used on the search bar and nowhere else. The designer chose this color to make the input pop based on the new design mockup, I haven't seen it being used anywhere else. My thought was that if he starts to use it, then I will make it a variable. What do you think?

Reply:
Hmm.  I saw it was only used in 1 spot as well, but I figured for consistency and ease of reuse, I would just make a variable and use that to plan ahead proactively.

Reply:
22c07d5

Reply:
Thanks @rdai10!